src/features/shared-link-settings-modal/SharedLinkSettingsModal.js

```diff
@@ -34,6 +63,9 @@ class SharedLinkSettingsModal extends Component {
          */
         onSubmit: PropTypes.func.isRequired,
 
+        // access level props
+        accessLevel: PropTypes.string,
```

Comment:
Should we be removing the `warnOnPublic` prop then? Also, do you think there's value in just passing in the whole `sharedLink` object to future proof this?

Reply:
oh yes, good call on removal. 

I thought about it, and I decided to add the one key value since the old modals (and this one) will eventually go away.